README.md

```diff
@@ -3224,6 +3229,425 @@ port-channel hash input offset. Valid values are integer or 'default'.
 port-channel symmetry hash. Valid values are true, false or 'default'.
 
 --
+### Type: cisco_route_map
+
+Manages a Cisco Route Map.
+
+| Platform | OS Minimum Version | Module Minimum Version |
+|----------|:------------------:|:----------------------:|
+| N9k      | 7.0(3)I2(1)        | 1.6.0                  |

```

Comment:
Did you go back all the way to verify this on `7.0(3)I2(1)`.  Since we are adding this feature in the `1.6.0` module it might be better to pick the latest I2 release as the OS Minimum Version.